Jenkinsfile

```diff
@@ -10,6 +10,7 @@ pipeline {
     }
 
     options {
+        disableConcurrentBuilds()
```

Comment:
@wanglinsong Why is the Jenkinsfile change in this PR? I don't see how they are related to this particular change.

Reply:
I have created a separate PR to update the Jenkins pipeline. https://github.com/prestodb/presto/pull/20233.